packages/auth/src/__tests__/auth.test.ts

```diff
@@ -1,6 +1,91 @@
-import Auth from '../index'
+import {rest} from 'msw'
+import {setupServer} from 'msw/node'
+import cookie from '@skillrecordings/cookies'
+import {ACCESS_TOKEN_KEY, USER_KEY} from '@skillrecordings/config'
 
-test('can create an instance', () => {
+import Auth, {getAuthorizationHeader, getLocalUser} from '../index'
+
+export const cornDogUser = {
+  name: 'Corn Dog',
+}
+
+export const server = setupServer(
+  // we can fake the actual user tat gets loaded with msw 🎉
+  rest.get(`/api/users/current`, (req, res, ctx) => {
+    return res(ctx.json(cornDogUser))
+  }),
+)
+
+beforeAll(() => server.listen())
+afterAll(() => server.close())
+
+afterEach(() => {
+  localStorage.removeItem(ACCESS_TOKEN_KEY)
+  cookie.remove(ACCESS_TOKEN_KEY)
+})
+
+test('can create an instance of Auth', () => {
   const auth = new Auth()
   expect(auth).not.toBeNull()
 })
+
+test('read the token from auth instance', async () => {
+  const auth = new Auth()
+  const accessToken = '0f95fab36e7a'
+
+  const token = await auth.eggheadAuth.token.getToken(
```

Comment:
this is exercising the [client-oauth2](https://www.npmjs.com/package/client-oauth2) token a bit while verifying that the `Auth` instance is initializing the client properly and it can be accessed